### PR TITLE
Catch and wrap with user-friendly message all unspecified Errors

### PR DESCRIPTION
**Pull request by dborzov**
When unspecified (programming) error is encountered, we output an error message like this that prompts the user to report it here as an issue:

Img

Before it was just spilling out the traceback as it is.

### CODE REVIEW

**Comment by JackMc** [comment]
Can we merge this? :smile:

File: datacats/error.py [hunk 1]
```diff
@@ -11,11 +11,11 @@ def __init__(self, message, format_args=(), parent_exception=None):
                 "type_description": parent_exception.user_description,
                 "message": parent_exception.__str__(),
             }
-            self.message = "".join(["{original}\n\n",
+            self.message = "".join([colored.blue("{original}\n\n").__str__(),
```
Comment (JackMc) [inline datacats/error.py]: Why not str(colored.blue(blah))?

Comment (dborzov) [inline datacats/error.py]: Good point! Tnx!

**Comment by dborzov** [comment]
Yeah, let's merge it! 
It includes turning off this annoying SSL warnings (when communicating with docker) snd that thing really needs to be on master.

**Comment by JackMc** [comment]
:+1:

File: datacats/error.py (hunk 1 above)
Comment (wardi) [inline datacats/error.py]: please merge after str() changes above and below

**Comment by wardi** [comment]
turning off the extra SSL warnings might be nice to backport to the current stable release too

**Pull request merged**